README.rst

```diff
@@ -21,44 +17,72 @@ The documentation for the development version is at
 
    http://statsmodels.github.io/dev/index.html
 
-
-This is currently a temporary solution that will be transformed into the new permanent location.
+**Note:** This is currently a temporary solution that will be transformed into the new permanent location.
 
 
 Main Features
 =============
 
-* linear regression models: Generalized least squares (including weighted least squares and
-  least squares with autoregressive errors), ordinary least squares.
-* glm: Generalized linear models with support for all of the one-parameter
+* Linear regression models:
+
+  - Ordinary least squares
+  - Generalized least squares
+  - Weighted least squares
+  - Least squares with autoregressive errors
```

Comment:
Quantile regression is missing, I'm not sure where it should go but we can put it under Linear regression models
`- Quantile regression`